search-app/test/cmr/search/test/api/tags.clj

```diff
@@ -0,0 +1,38 @@
+(ns cmr.search.test.api.tags
+  "Tests to verify functionality in cmr.search.api.tags namespace."
+  (:require
+   [clojure.test :refer :all]
+   [cmr.search.api.tags :as tags]))
+
+(def dictionary-with-error [
+    {
+        "errors" [
+        "Collection [C1200000006-PROV1] does not exist or is not visible."
+        ]
+
+        "tagged_item" {
+        "concept_id" "C1200000006-PROV1"
+        }
+    }
+])
+
+(def dictionary-with-no-error [
+    {
+        "tag_association" {
+            "concept_id" "TA1200000009-CMR"
+            "revision_id" 1
+        }
+        "tagged_item" {
+            "concept_id" "C1200000005-PROV1"
+        }
+    }
+])
+
+(deftest tag-api-response-status-code-test
+  (testing "Making sure `tag-api-response` returns status code 400 when there's errors"
+      (do
```

Comment:
Unnecessary `do`, the test runner will handle both `is` forms without the extra wrapper. And the `=` can be on the same line as the `is`